src/openzaak/utils/filterset.py

```diff
@@ -15,3 +16,12 @@ def filter_for_field(cls, field, field_name, lookup_expr=None):
         if not filter.extra.get("help_text"):
             filter.extra["help_text"] = getattr(field, "help_text", None)
         return filter
+
+
+class OrderingFilter(_OrderingFilter):
+    def filter(self, qs, value):
+        if value in constants.EMPTY_VALUES:
+            return qs
+
+        ordering = [self.get_ordering_value(param) for param in value]
+        return qs.order_by(*ordering).distinct()
```

Comment:
Do you really need `distinct` here? It can degrade our performance

Reply:
Yes we need distinct here because the original queryset already is ordered with distinct. Using order_by without it will result in errors.